dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporterTest.java

```diff
@@ -183,14 +723,52 @@ public void testWithLabels() throws IOException {
             .setUrl(connection.getURL())
             .build();
 
-    CompletableResultCode result = metricExporter.export(Collections.singleton(md), connection);
+    InMemoryDtMetricReader reader = new InMemoryDtMetricReader(metricExporter);
+    SdkMeterProvider sdkMeterProvider =
+        SdkMeterProvider.builder()
+            .setClock(testClock)
+            .registerMetricReader(reader)
+            .registerView(
+                InstrumentSelector.builder()
+                    .setInstrumentType(InstrumentType.COUNTER)
+                    .setInstrumentName("testSum")
+                    .build(),
+                View.builder()
+                    .setAggregation(Aggregation.sum(AggregationTemporality.DELTA))
+                    .build())
+            .build();
 
-    verify(connection).setRequestMethod("POST");
-    verify(connection).setRequestProperty("Authorization", "Api-Token mytoken");
-    verify(connection).setRequestProperty("Content-Type", "text/plain; charset=utf-8");
+    Meter sdkMeter = sdkMeterProvider.get(getClass().getName());
+    LongCounter counter = sdkMeter.counterBuilder("testSum").build();
+
+    counter.add(100);
+    sdkMeterProvider.forceFlush();
+    testClock.advance(Duration.ofSeconds(1));
+
+    counter.add(200);
+    testClock.advance(Duration.ofSeconds(1));
+    CompletableResultCode result = sdkMeterProvider.forceFlush();
+
+    assertTrue(result.isSuccess());
     assertEquals(
-        "name,dt.metrics.source=opentelemetry,label1=val1,label2=val2 count,delta=194.0 1619687659000",
+        "testSum,dt.metrics.source=opentelemetry count,delta=100 1557212400000testSum,dt.metrics.source=opentelemetry count,delta=200 1557212402000",
```

Comment:
These are not separated at all

Reply:
Same as above. Now each line is asserted individually.